### ISSUE TITLE
Use dog without a .dogrc

### ISSUE THREAD

**Issue by teran-mckinney** [DataDog/datadogpy#107]
Hi,

Is there a way to use `dog` without a dogrc? Ideally, to steal the `api_key` from `/etc/dd-agent/datadog.conf`?

I haven't had any luck with this hack:

```
dog --api-key $(grep api_key /etc/dd-agent/datadog.conf | head -n 1 | awk '{print $NF}') event post --tags deleteme 'test string'
```

Goal is to make sure crons are running successfully.

Thank you,
Teran

**Comment by yannmh** [comment]
Hi @teran-mckinney,

Sorry to hear that you are experiencing issues with the `dog` command. 

I had a look and you are completely right: the `dog` should be usable without a `.dogrc` file but an API key specified in the command line parameters. 
Currently this behavior is only achievable when both optional parameters `--api-key` and `--application-key` are specified. 

I'll work on a fix for the next release. In the meantime, a simple workaround would be to specify any string  as an application key (it does not need to be valid as long as `dog` is used to "write" data).
To sum up, this is how your command line could look like:
```
dog --api-key $(grep api_key /etc/dd-agent/datadog.conf | head -n 1 | awk '{print $NF}') --application-key none event post --tags deleteme 'test string'
```

**Comment by willianpaixao** [comment]
Can I take this issue?

**Comment by irabinovitch** [comment]
@willianpaixao That would be fantastic.  Thanks!

**Comment by zippolyte** [comment]
We are not planning on reading API keys from the Datadog Agent configuration file, but we do support reading the usual DD_API_KEY and DD_APP_KEY environment variables.
